specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2019-01-01/apimapis.json

```diff
@@ -2296,7 +2296,7 @@
             "in": "body",
             "required": true,
             "schema": {
-              "$ref": "./definitions.json#/definitions/SchemaContract"
+              "$ref": "./definitions.json#/definitions/SchemaCreateOrUpdateContract"
```

Comment:
Updating the name here may cause a breaking change in the SDKs since the name of the type representing this definition is changing. Do you need to update this one? or should the update to "document" property be happening in the SchemaContract definition?

Reply:
For CRUD on `Schema`, the CreateOrUpdate contract different from the response `SchemaContract`
for CreateOrUpdate the contract requires `properties.document.value` but the response is `properties.document`
In existing clients this scenario was broken, hence we need to split the contract, otherwise, when the resource is created client cannot serialize the `document` in the response

Reply:
we can make `document` as `jobject`, but it will degrade the experience when writing clients. Since the scenario anyway was not working, I think creating a good client experience is fine.